src/shared/lib/AnnotationUtils.ts

```diff
@@ -35,7 +35,8 @@ export function recurrentHotspotFilter(hotspot: Hotspot) {
     // only single and indel mutations are regular hotspots
     return (
         hotspot.type.toLowerCase().includes('single') ||
-        hotspot.type.toLowerCase().includes('indel')
+        hotspot.type.toLowerCase().includes('indel') ||
+        hotspot.type.toLowerCase().includes('splice')
```

Comment:
This function only been used in getting oncoprint hotspots data (https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/resultsView/ResultsViewPageStore.ts#L4283), not sure if I should add `splice` as one of the filter type?

Reply:
This should be okay, assuming that splice hotspots are recurrent hotspots. I think this filter was added mainly to exclude 3d hotspots.

Reply:
This is fixed now. We need to apply the same "splice logic" to oncoprint hotspot data